### ISSUE TITLE
New Help Text for Accordion Box 

### ISSUE THREAD

**Issue by Matthew-Moore240** [phetsims/ratio-and-proportion#189]
This is the new help text for the Accordion Box and should be available only if the Accordion Box  is expanded.

**The new help text should read:** 
Choose left hand and right hand values to set your ratio challenge.

**Comment by Matthew-Moore240** [comment]
Looks good! Closing.
